modular_tannhauser/modules/CitOwOChems/code/chemistry/reagents/furrrrrr.dm

```diff
@@ -47,14 +46,6 @@
 			if(prob(10))
 				to_chat(M, "You find yourself unable to supress the desire to howl!")
 				M.emote("awoo")
-			if(prob(20))
```

Comment:
Broken or removing for other reasons?

Reply:
```
var/list/datum/weakref/seen = list(ismob(0))
for(var/victim in seen)
```
This piece at the top means nothing will happen, even if the prob gets rolled, since it's looping through an empty list
I could fix it up so it works, if the bulge-noticing isn't too ToS for our server

Reply:
yeah lets fix this section up

Reply:
Should be working now
![image](https://user-images.githubusercontent.com/47021791/180911431-d3fa0c2b-50ad-4f4e-a259-86a78f0edcf8.png)